src/test/cli/crushtool/add-item.t

```diff
@@ -1,5 +1,9 @@
   $ crushtool -i "$TESTDIR/simple.template" --add-item 0 1.0 device0 --loc host host0 --loc cluster cluster0 -o one > /dev/null
   $ crushtool -i one --add-item 1 1.0 device1 --loc host host0 --loc cluster cluster0 -o two > /dev/null
+  $ crushtool -i two --create-simple-rule simple-rule cluster0 host firstn -o two > /dev/null
+  $ crushtool -d two -o final
```

Comment:
I mean, leave of -o final, so that the output appears here, and therefore if we ever break the command(s) this test will fail to match the output.